src/components/Form/Debloat.js

```diff
@@ -0,0 +1 @@
+export function Debloat() {}
```

Comment:
We have to trick react-styleguidist to render the component and so we have to put this here